pyiceberg/io/pyarrow.py

```diff
@@ -1720,13 +1720,22 @@ def write_file(table: Table, tasks: Iterator[WriteTask]) -> Iterator[DataFile]:
     except StopIteration:
         pass
 
+    compression_codec = table.properties.get("write.parquet.compression-codec")
+    compression_level = table.properties.get("write.parquet.compression-level")
+    if compression_codec == "uncompressed":
+        compression_options = {"compression": "none"}
+    else:
```

Comment:
How do you feel about checking if `compression_codec is not None` before adding the compression option to the ParquetWriter kwarg?

I think we can use the meaningful distinction between None and string 'NONE' to mean either that the compression codec was not set (and hence should use the default) versus 'NONE' meaning that we'd like to avoid compression altogether. I think this might help clean up some of the if conditions in **test_parquet_compression**

Reply:
Sorry I don't understand what you mean

Reply:
Sorry for the confusion. I was trying to find an alternative to using the string "<not set>" to emulate the case where write.parquet.compression-codec isn't set in the test case below.

Reply:
Sorry I still don't understand what you want me to do

Reply:
Let's resolve this review comment. Your review recent commit fixed the pattern in the test, and we shouldn't expect to see **None** table property values here.